docs/asyncapi.yaml

```diff
@@ -513,9 +448,9 @@ components:
       example: G00 - Gross value of claim
     DueDate:
       type: string
-      format: DD/MM/YYYY, YYYY/MM/DD, DD-MM-YYYY or YYYY-MM-DD
+      format: 'DD/MM/YYYY, YYYY/MM/DD, DD-MM-YYYY or YYYY-MM-DD'
```

Comment:
Does this need the single quotes?

Reply:
It does not and this is also incorrect too.  Corrected.